appendices/contributors/text.xml

```diff
@@ -23,7 +23,7 @@ This page lists the contributions to the Gentoo Development Guide:
 </author>
 <author name="Tom Martin" email="slarti@gentoo.org">
   <uri link="::appendices/editor-configuration/emacs"/>,
-  <uri link="::ebuild-writing/functions/src_unpack/svn-sources"/>,
+  Subversion Sources (removed),
```

Comment:
Any reason why we wouldn't just remove the reference to the specific contribution that doesn't exist anymore? If we do it this way, the list will forever grow and become a mess, no?

Reply:
No strong opinion here. Let's remove it.